packages/mux-player/test/template.test.js

```diff
@@ -11,7 +11,7 @@ describe('<mux-player> template render', () => {
     render(content({}), div);
     assert.equal(
```

Comment:
non-blocking: If we end up settling on these attributes passed into themes, we should add more tests around how they behave when various values are set/omitted, including over time.